src/pybind/mgr/cephadm/services/monitoring.py

```diff
@@ -301,6 +303,13 @@ def generate_config(
                 'url': build_url(host=addr, port=port).lstrip('/')
             })
 
+        # scrape ceph exporters
+        for dd in self.mgr.cache.get_daemons_by_service('ceph-exporter'):
+            assert dd.hostname is not None
+            deps.append(dd.name())
+            addr = self._inventory_get_fqdn(dd.hostname)
+            mgr_scrape_list.append(build_url(host=addr, port=exporter_port).lstrip('/'))
```

Comment:
`mgr_scrape_list` is meant to be the list of host/port for the mgr daemons. Can we use a different variable for gathering the list of ceph-exporters?

Reply:
OK, sure!